examples/browser/package.json

```diff
@@ -19,17 +19,12 @@
     "@theia/console": "^0.14.0",
     "@theia/core": "^0.14.0",
     "@theia/debug": "^0.14.0",
-    "@theia/debug-nodejs": "^0.14.0",
     "@theia/editor": "^0.14.0",
     "@theia/editor-preview": "^0.14.0",
-    "@theia/editorconfig": "^0.14.0",
     "@theia/file-search": "^0.14.0",
     "@theia/filesystem": "^0.14.0",
     "@theia/getting-started": "^0.14.0",
     "@theia/git": "^0.14.0",
-    "@theia/java": "^0.14.0",
-    "@theia/java-debug": "^0.14.0",
-    "@theia/json": "^0.14.0",
```

Comment:
we should land https://github.com/eclipse-theia/theia/pull/6647 or keep using it for now

Reply:
Sounds good, I left it for now.